### PR TITLE
Global Furnaces+LowTempRadiantSystems+ScheduleManager

### PR DESCRIPTION
**Pull request by jmythms**
Pull request overview
---------------------
Trying to merge in globals from 
 - Furnace.cc, .hh, .unit.cc. 
 - LowTempRadiantSystem.cc, .hh, 
 	 - Also formatted code in .hh file
 - ScheduleManager.cc, .hh, 

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/LowTempRadiantSystem.cc [hunk 1]
```diff
@@ -5130,7 +5130,6 @@ namespace LowTempRadiantSystem {
         using DataHVACGlobals::TimeStepSys;
 
         // SUBROUTINE PARAMETER DEFINITIONS:
-        auto constexpr RoutineName("UpdateLowTempRadiantSystem");
```
Comment (jmythms) [inline src/EnergyPlus/LowTempRadiantSystem.cc]: These RoutineName-s are typically used in error strings, but if this routine does not have error messages, is it necessary to keep them?

**Comment by Myoldmopar** [comment]
OK, I pulled in latest develop and rebuilt after resolving a couple minor conflicts.  I ran unit tests, the full ctest suite, and then ran ~300 regressions, all with no issues or diffs.  I am going to push now and merge right away to keep the CI cycles away from this.  

@jmythms thank you for your effort here, this is a big one!

**Comment by Myoldmopar** [comment]
(I cancelled the Alternate Linux Builds so you may get warnings about that, just ignore)

**Comment by jmythms** [comment]
> OK, I pulled in latest develop and rebuilt after resolving a couple minor conflicts. I ran unit tests, the full ctest suite, and then ran ~300 regressions, all with no issues or diffs. I am going to push now and merge right away to keep the CI cycles away from this.
> 
> @jmythms thank you for your effort here, this is a big one!

Thank you!! This was fun.

**Pull request merged**